tubular/edx_api.py

```diff
@@ -245,7 +286,9 @@ def retirement_retire_mailings(self, learner):
         """
         params = {'data': {'username': learner['original_username']}}
         with correct_exception():
-            return self._client.api.user.v1.accounts.retire_mailings.post(**params)
+            api_url = self.get_api_url('api/user/v1/accounts/retire_mailings')
+            results = self._request('POST', api_url, data=params).json()
```

Comment:
Please, confirm that the `data` should be used here to request, not the `params`. Just to double-check

Reply:
@dyudyunov  i cannot find declaration of this method in api lms
and i think about this handler is excess